02-props-state-component-architecture/props/pokedex/src/pokedex.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import Pokecard from './pokecard';
+
+
+class Pokedex extends Component{
+	render(){
+		const pokemoni = this.props.pokemon.map(pokemon => {
+			return (
+			<Pokecard
+				key={pokemon.id}
+				name={pokemon.name}
+				type={pokemon.type}
+				image={pokemon.image} 
```

Comment:
this works, but in the future you can use the rest operator for when the prop names are the same as the pokemon keys:

```jsx
const { id, ...rest } = pokemon;
<Pokecard
   key={id}
   {...rest}
 />
```